native/core/src/parquet/parquet_support.rs

```diff
@@ -1861,6 +1864,40 @@ fn trim_end(s: &str) -> &str {
     }
 }
 
+#[cfg(not(feature = "hdfs"))]
+pub(crate) fn register_object_store(
+    session_context: Arc<SessionContext>,
+) -> Result<(), ExecutionError> {
+    let object_store = object_store::local::LocalFileSystem::new();
+    let url = Url::try_from("file://").unwrap();
+    session_context
+        .runtime_env()
+        .register_object_store(&url, Arc::new(object_store));
+    Ok(())
+}
+
+#[cfg(feature = "hdfs")]
+pub(crate) fn register_object_store(
+    session_context: Arc<SessionContext>,
+) -> Result<(), ExecutionError> {
+    // TODO: read the namenode configuration from file schema or from spark.defaultFS
+    let url = Url::try_from("hdfs://namenode:9000").unwrap();
```

Comment:
this will be addressed in https://github.com/apache/datafusion-comet/issues/1360